rhombus/rhombus/scribblings/ref-comparable.scrbl

```diff
@@ -11,9 +11,9 @@
 A @deftech{comparable} value is one that supports @rhombus(<),
 @rhombus(<=), @rhombus(>=),@rhombus(>=), @rhombus(compares_equal), and
 @rhombus(compares_unequal). Real numbers, @tech{characters},
-@tech{strings}, @tech{byte strings}, @tech{symbols}, and @tech{keywords}
-are all comparable, as are instances of classes that implement
-@rhombus(Comparable, ~class).
+@tech{strings}, @tech{byte strings}, @tech{symbols}, @tech{keywords},
+and paths are all comparable, as are instances of classes that
```

Comment:
Could you add a simple description (we can expand it later) to “Paths” along with a `deftech` so that this link would work?  Something like `A @deftech{path} value represents a filesystem path`.